### ISSUE TITLE
Improve accessibility by using higher contrast on the website.

### ISSUE THREAD

**Issue by GabLotus** [bevyengine/bevy-website#3]
In the spirit of accessibility we should be careful to not use difficult to read color formats/palettes on the website.

**Comment by pppKin** [comment]
Can we also have light/dark mode switch? I have trouble reading texts on dark background and have always used in-browser reader on bevy websites. However the browser (Readed View in firefox at least) tend to mess up original page layout,  making it difficult to navigate through the article.
